### PR TITLE
Ensure stdcm form and simulations clear on page refresh

### LINKED ISSUE #9754: LMR Refreshing the page completely clears the form and any simulations below.

### PR DESCRIPTION
**Pull request by achrafmohye**
Closes #9754

### CODE REVIEW

**Comment by emersion** [comment]
Curious, I would've expected that to already be the case, seeing how `stdcmConfSlice` is in the blacklist in `persistConfig`.

**Review by Akctarus — APPROVED** [review]
Tested, lgtm !

File: front/src/reducers/osrdconf/stdcmConf/index.ts
Comment (SharglutDev) [inline front/src/reducers/osrdconf/stdcmConf/index.ts:1]: When I refresh the page, the fields for maxSpeed, length and mass are not empty

Comment (achrafmohye) [reply front/src/reducers/osrdconf/stdcmConf/index.ts:1]: i think after the merge of @Wadjetz pr, i'll push a commit that resets the consist form values

Comment (achrafmohye) [reply front/src/reducers/osrdconf/stdcmConf/index.ts:1]: Done !

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested (don't forget to fixup your commits)

**Pull request merged**